src/parallel_som.jl

```diff
@@ -549,5 +516,40 @@ function convertTrainingData(data)::Array{Float64,2}
     return train
 end
 
+function splitMatrix(matrix_train, nWorkers)
```

Comment:
May you add the docstring for this function?

Reply:
removed function